plugins/MauticMessengerBundle/Controller/MessengerController.php

```diff
@@ -0,0 +1,621 @@
+<?php
```

Comment:
Take a look at the ChannelBundle and try to leverage  AbstractStandardFormController https://github.com/mautic/mautic/blob/c0c9e27b511a369f4f81d9b94cdc9e7eb74e678d/app/bundles/ChannelBundle/Controller/MessageController.php#L24-L24.

We'd like to move away from all this copy/paste duplicate code in the controllers. It will eliminate a ton of code.